templates/vue/src/components/tyde/activities/CircleOfSupport/CommunityView/index.vue

```diff
@@ -15,36 +15,99 @@
       @add-connection="$emit('add-connection', $event)"
       @edit-connection="handleEditConnection"
       @add-community="$emit('add-community', $event)"
+      @connection-submitted="handleConnectionSubmitted"
+      @connection-closed="handleConnectionClosed"
     />
     <add-community-tab
       v-model="community"
       :show="isCommunityTabOpen"
       :disabled="!canAddCommunity"
       @back="state = lastState"
-      @add-community="$emit('add-community', $event)"
+      @add-community="obHandleAdded($event)"
       @toggle="toggleCommunityTab"
     />
+    <welcome-communities
+      v-if="isState('Communities.Welcome')"
+      class="welcome-communities"
+      @continue="handleContinue"
+    />
+    <welcome-connections
+      v-if="isState('Connections.Welcome')"
+      class="welcome-connections"
+      @continue="handleContinue"
+    />
+    <add-confirmation
+      v-if="isState('Communities.AddMoreConfirmation')"
+      @later="send(OnboardingEvents.AddLater)"
+      @another="send(OnboardingEvents.AddAnother)"
+    />
+    <ob-finish-view
+      v-if="isState('Connections.Finish')"
+      :connections="connections"
+      @ob-finish="send(OnboardingEvents.Done)"
+    />
+    <tooltip v-if="isState('Communities.AddLaterTooltip')" position="right">
+      <h3>
+        Remember - you can click this button whenever you'd like to add another
+        community!
+      </h3>
+      <b-button pill variant="secondary" @click="send(OnboardingEvents.Continue)">
+        Got it &#8594;
+      </b-button>
+    </tooltip>
+    <tooltip v-if="isState('Communities.AddAnotherTooltip')" position="right">
+      <h3>
+        Click here to add another community!
+      </h3>
+      <b-button pill variant="secondary" @click="send(OnboardingEvents.Add)">
+        Got it &#8594;
+      </b-button>
+    </tooltip>
+    <tooltip v-if="isState('Connections.AddAnotherTooltip')" position="left">
+      <h3>
+        Click here to add some
+        <br />
+        of your connections!
+      </h3>
+      <b-button pill variant="secondary" @click="send(OnboardingEvents.Add)">
+        Got it &#8594;
+      </b-button>
+    </tooltip>
```

Comment:
Since these are all related to the onboarding process, I suggest we move all of this and whatever onboarding logic or code we can to a separate component to keep this file clean and minimal.